Music app/app/src/main/java/com/example/musicapp/Activities/AlbumDetails.java

```diff
@@ -92,6 +92,7 @@ public void moveItem(int oldPos, int newPos)
         MusicFile musicFile = MainActivity.musicFiles.get(oldPos);
         MainActivity.musicFiles.remove(oldPos);
         MainActivity.musicFiles.add(newPos, musicFile);
-        albumDetailsAdapter.notifyItemMoved(oldPos, newPos);
+        if (albumDetailsAdapter != null)
```

Comment:
why this is necessary?

Reply:
It can't be uninitialized in-app logic, but it is necessary for unit testing.